### ISSUE TITLE
Add installation instructions for school branch / binder links

### ISSUE THREAD

**Issue by sdrave** [pymor/2020_online_course#1]

**Comment by renefritze** [comment]
I am somewhat confident that I'll be able to produce conda packages via azure now:
https://dev.azure.com/pymor/pymor-feedstock/_build/results?buildId=3493&view=logs&j=9c5ef928-2cd6-52e5-dbe6-9d173a7d951b

**Comment by renefritze** [comment]
I managed to get conda packages for all platforms: https://github.com/pymor/2020_online_course/tree/main/packages/conda
Conda packages are untested, but built from the same recipe as the release packages, on (mostly) the same azure infra.

I also added manylinux wheels: https://github.com/pymor/2020_online_course/tree/main/packages/wheels

**Comment by renefritze** [comment]
@pymor/pymor-devs  I've just triggered a new build for the conda packages here: https://dev.azure.com/pymor/pymor-feedstock/_build/results?buildId=3496&view=results
with more packages: https://github.com/pymor/pymor-feedstock/commit/34508783f482a99b1c74d43c89bf4fd1f0c06c41

**Comment by renefritze** [comment]
I just realized we could also pretty easily add a docker-based setup for people to work from.

**Comment by pmli** [comment]
Ok, it seems conda's dependency resolver has an issue with this conda package. Whenever I try to install something after installing the pyMOR package, it raises an `UnsatisfiableError`.

I'm not sure about using docker...

I also found an issue with binder. It seems to install an older version of pyMOR which doesn't have `formatsrc` or `bode_plot`.

**Comment by renefritze** [comment]
Can your install command pull dependencies from conda-forge? The package ist build using their infrastructure setup.

**Comment by renefritze** [comment]
@pmli Good catch for the pyMOR version. I've fixed that:  https://github.com/pymor/2020_online_course/commit/51ac6d45cd4f5da8c0f23ba14816f25fbbb34921

**Comment by TiKeil** [comment]
I just tried to conda install the packages on my Mac and it seems that I also have package inconsistencies. For some reason, some packages disappear almost randomly. However I found a workaround how to still use conda for installing pymor. 

```
conda install -c conda-forge diskcache
conda install pymor-2020_online_course-py38h8c46c48_0.tar.bz2
conda install -c conda-forge numpy
```

seems to work in a fresh conda env. Don't ask me why and how to do this in a better way.

**Comment by renefritze** [comment]
Added docker setup in #3

**Comment by pmli** [comment]
I have some issue with matplotlib when running `visualize` in getting_started. It complains about the backend, but tutorial_bt works fine.

**Comment by renefritze** [comment]
> I have some issue with matplotlib when running `visualize` in getting_started. It complains about the backend, but tutorial_bt works fine.

In the binder setup?

**Comment by pmli** [comment]
> > I have some issue with matplotlib when running `visualize` in getting_started. It complains about the backend, but tutorial_bt works fine.
> 
> In the binder setup?

On Windows.

**Comment by renefritze** [comment]
Can you identify if it's a matter of installing an additional backend?

**Comment by pmli** [comment]
I'm not sure how to install an additional backend... I tried by installing `Qt.py`, but that didn't help...

**Comment by renefritze** [comment]
The backed would be pyqt. If you look at https://github.com/pymor/pymor-feedstock/blob/2020_online_course/recipe/meta.yaml#L43 this should already be installed as a dependency of the pyMOR conda package.
